src/main/java/net/preibisch/mvrecon/fiji/spimdata/imgloaders/OMEZarrAttibutes.java

```diff
@@ -130,23 +130,10 @@ public static OmeNgffMultiScaleMetadata[] createOMEZarrMetadata(
 		return meta;
 	}
 
-	// TODO: this is inaccurate, we should actually estimate it from the final transformn that is applied
-	public static double[] getResolutionS0( final VoxelDimensions vx, final double anisoF, final double downsamplingF )
+	// The method assumes that the voxel dimensions are at scale S0, so it simply returns the dimensions as double array.
+	public static double[] getResolutionS0( final VoxelDimensions vx )
```

Comment:
Changed the signature here because now the anisotropy and the downsampling factors are not used. These can be added when they are actually used

Reply:
@StephanPreibisch when I tested the voxel dimensions were already set to whatever I set in the UI. The use case implemented before I removed it was to set  the Z-resolution to X-resolution if anisotropy was not set, but that creates the possibility of overriding the Z-resolution to a different value than the one set by the user. That's why I said I couldn't see how using the anisotropy would help, since users can simply set the calibration values to whatever they want.